### DISCUSSION TITLE
Is there any way to dismissScreen in specific Screen for iOS 15

### DISCUSSION THREAD

**Discussion by ajinumoto** [SwiftfulThinking/SwiftfulRouting#55] [Feature Requests]
First thanks Nick for creating this library! 

In my project i have

`FirstScreen -> SecondScreen -> ThirdScreen -> MainScreen`

Lets say in the `MainScreen` there are some button with action

```
            Button("Submit") {
                if isFailed {
                    router.dismissScreen(FirstView.self)
                } else {
                    router.dismissScreen(SecondView.self)
                }
            }
```

How can achieve this for iOS 15? Thank you

**Comment by nagarjuna-techension** [comment]
Hi,

 I need to redirect to root view. Please suggest if you solve this problem. Thank you.

**Comment by aryanarayantiwari** [comment]
@SwiftfulThinking any update on this ?

**Comment by SwiftfulThinking** [comment]
In version 6.1 you can now dismiss to sheet by id. There are several new dismiss methods, such as dismiss(to: "screen_1") or dismiss(count: 2)
